packages/jest-mock/__typetests__/utility-types.test.ts

```diff
@@ -105,64 +105,63 @@ type IndexObject = {
   propertyB: {b: string};
 };
 
-// ClassLike
+test('ClassLike', () => {
+  expect<ClassLike>().type.toBeAssignable(SomeClass);
 
-expectAssignable<ClassLike>(SomeClass);
-expectNotAssignable<ClassLike>(() => {});
-expectNotAssignable<ClassLike>(function abc() {
-  return;
+  expect<ClassLike>().type.not.toBeAssignable(() => {});
+  expect<ClassLike>().type.not.toBeAssignable(function abc() {
+    return;
+  });
+  expect<ClassLike>().type.not.toBeAssignable('abc');
+  expect<ClassLike>().type.not.toBeAssignable(123);
+  expect<ClassLike>().type.not.toBeAssignable(false);
+  expect<ClassLike>().type.not.toBeAssignable(someObject);
+});
+
+test('FunctionLike', () => {
+  expect<FunctionLike>().type.toBeAssignable(() => {});
+  expect<FunctionLike>().type.toBeAssignable(function abc() {
+    return;
+  });
+
+  expect<FunctionLike>().type.not.toBeAssignable('abc');
+  expect<FunctionLike>().type.not.toBeAssignable(123);
+  expect<FunctionLike>().type.not.toBeAssignable(false);
+  expect<FunctionLike>().type.not.toBeAssignable(SomeClass);
+  expect<FunctionLike>().type.not.toBeAssignable(someObject);
+});
+
+test('ConstructorKeys', () => {
+  expect<ConstructorLikeKeys<OptionalInterface>>().type.toEqual<
+    'constructorA' | 'constructorB'
+  >();
+  expect<ConstructorLikeKeys<SomeObject>>().type.toEqual<'SomeClass'>();
 });
-expectNotAssignable<ClassLike>('abc');
-expectNotAssignable<ClassLike>(123);
-expectNotAssignable<ClassLike>(false);
-expectNotAssignable<ClassLike>(someObject);
 
-// FunctionLike
+test('MethodKeys', () => {
+  expect<MethodLikeKeys<SomeClass>>().type.toEqual<'methodA' | 'methodB'>();
```

Comment:
Two types are compared directly. No need of `declare const` in utility type tests. Also nice improvement.